mmdet3d/models/backbones/h3d_backbone.py

```diff
@@ -26,13 +26,12 @@ def __init__(self, backbone=None, **kwarg):
             out_channel += bb_cfg['fp_channels'][-1][-1]
             self.backbone_list.append(build_backbone(bb_cfg))
 
-        ### Feature concatenation
-        self.conv_agg1 = torch.nn.Conv1d(out_channel, out_channel//2, 1) 
-        self.bn_agg1 = torch.nn.BatchNorm1d(out_channel//2)
-        self.conv_agg2 = torch.nn.Conv1d(out_channel//2, 256,1)
+        # Feature concatenation
+        self.conv_agg1 = torch.nn.Conv1d(out_channel, out_channel // 2, 1)
```

Comment:
1. use `ConvModule` in `mmcv.cnn` instead of bn(conv(....)).
2. `out_channel` should be initialized at `__init__()`.